squarelet/templates/account/email/email_confirmation_signup_message.html

```diff
@@ -10,45 +10,66 @@
   {% url "account_set_password" as url_password %}
   {% autologin url_password user as url_password_ %}
   {% autologin activate_url user as activate_url_ %}
-  {% blocktrans with username=user.username %}
-    <p>
-    Thank you for creating an account on MuckRock.
-    </p>
-    <p>
-    Your username is
-    <a href="{{ url_user_ }}">
-      {{ username }}
-    </a>.
-    </p>
+
+  {% blocktrans with full_name=user.full_name %}
+    <p>Welcome to your MuckRock account, {{ full_name }}!</p>
+  {% endblocktrans %}
```

Comment:
Should also fall back to `username` in the same way as at the bottom:
```suggestion
  {% blocktrans with full_name=user.full_name username=user.username %}
    <p>Welcome to your MuckRock account, {{ full_name|default:username }}!</p>
  {% endblocktrans %}
```